src/routes/docs/advanced/platform/otp-sms/+page.markdoc

```diff
@@ -0,0 +1,217 @@
+---
+layout: article
+title: OTP SMS
+description: Learn how Appwrite handles OTP and SMS-based authentication for secure user verification.
+---
+
+{% info title="Note" %}
+Changes will take effect on February 10th, 2025.
+{% /info %}
+
+Appwrite supports OTP (One-Time Password) and SMS-based authentication to provide secure and reliable user verification. This feature enhances your app's security by adding an extra layer of authentication.
+
+# Free messages {% #free-messages %}
+
+All Appwrite plans include **10 free SMS messages** per month, which allows you to test and implement OTP functionality without immediate costs.
+
+# Additional messages {% #additional-messages %}
+
+To send more than 10 SMS messages per month, you need to upgrade to a paid plan. You can find more information about the available plans [here](https://appwrite.io/pricing).
+
+The cost for additional messages is calculated based on two factors:
+1. The number of messages sent
+2. The destination country of each message
+
+Appwrite works with various telecom providers to reduce SMS rates as much as possible in an effort to make it affordable for everyone.
+
+## Country-specific rates {% #country-rates %}
+
+SMS rates vary by country due to differences in telecom infrastructure and regulations. Here is a breakdown of the rates:
+
+| Country code | Country name                                                 | Price per SMS |
+|--------------|--------------------------------------------------------------|---------------|
+| +213         | Algeria                                                      | $ 0.1885      |
```

Comment:
Rates below seems incorrect, please update the CLI which we used to generate the numbers.